xblock/VERSION.txt

```diff
@@ -1 +1 @@
-1.3.1
+1.4
```

Comment:
nit: this should be `1.4.0`

Reply:
@xitij2000 Done. Should I ask natabene for review?